src/views/protected/CoachingResourcesViews/LiteracyInstruction.tsx

```diff
@@ -10,8 +10,8 @@ import {
 import ResourcePageLayout from './ResourcePageLayout'
 
 import LiteracyInstructionIcon from '../../../assets/images/LiteracyIconImage.svg'
-import HandoutDocumentUrl from '../../../assets/coaching-docs/Literacy Definitions and Example.pdf'
-import PresentationUrl from '../../../assets/coaching-docs/Literacy Instruction CHALK Presentation.pptx'
+import HandoutDocumentUrl from '../../../assets/coaching-docs/Literacy Definitions and Examples.pdf'
+// import PresentationUrl from '../../../assets/coaching-docs/Literacy Instruction CHALK Presentation.pptx'
```

Comment:
All the commented out code is redundant - in case we want to bring it back, VCS will help us. Let's remove all the commented out code from the file
```suggestion
```